src/main/resources/application.yml

```diff
@@ -83,10 +83,15 @@ interventions-ui:
     sent-referral: "/service-provider/referrals/{id}/details"
     submit-action-plan: "/service-provider/referrals/{id}/progress" # fixme
     appointment-not-attended: "/service-provider/referrals/{id}/progress" # fixme
+    view-appointment: "/probation-practitioner/referrals/{id}/progress"
```

Comment:
i have added this location in another PR and called it `pp-referral-progress` - can we call it that here too to make the future merge clean?